eth/tools/builder/chain/builders.py

```diff
@@ -325,22 +298,12 @@ def disable_pow_check(chain_class: Type[ChainAPI]) -> Type[ChainAPI]:
         blocks mined this way will not be importable on any chain that does not
         have proof of work disabled.
     """
-    if not chain_class.vm_configuration:
-        raise ValidationError("Chain class has no vm_configuration")
-
-    if issubclass(chain_class, NoChainSealValidationMixin):
-        # Seal validation already disabled, hence nothing to change
-        chain_class_without_seal_validation = chain_class
-    else:
-        chain_class_without_seal_validation = type(
-            chain_class.__name__,
-            (chain_class, NoChainSealValidationMixin),
-            {},
-        )
-    return chain_class_without_seal_validation.configure(  # type: ignore
-        vm_configuration=_mix_in_disable_seal_validation(
-            chain_class_without_seal_validation.vm_configuration  # type: ignore
-        ),
+    no_pow_vms = tuple(
+        (block_number, vm.configure(consensus_class=NoProofConsensus))
+        for block_number, vm in chain_class.vm_configuration
+    )
+    return chain_class.configure(
+        vm_configuration=no_pow_vms,
```

Comment:
Yeah, this seems like a nice place to use that `SimpleConsensusApplier` concept.